app/assets/stylesheets/shared/sizes.scss

```diff
@@ -0,0 +1,2 @@
+$padding_topic_posts: 8px 16px 8px 16px;
```

Comment:
you don't need to save all of the sizes inside the variables. Set this padding directly in the class where you've used this variable and remove this variable and the variable below